packages/code-du-travail-data/dataset/themes/themes.json

```diff
@@ -1363,8 +1363,8 @@
   },
   {
     "id": 229,
-    "label": "Congé de formation - Animateur ou cadre pour la jeunesse",
-    "slug": "229-conge-de-formation-animateur-ou-cadre-pour-la-jeunesse",
+    "label": "Congé de formation / Animateur ou cadre pour la jeunesse",
+    "slug": "229-conge-de-formation-/-animateur-ou-cadre-pour-la-jeunesse",
```

Comment:
I'm not sure introducing `/` in a url is a good idea
`-` is ok as long a we can split theme